include/heap.h

```diff
@@ -28,7 +28,7 @@ THUMB_FUNC void FUN_02016988();
 void *AllocFromHeap(u32 heap_id, u32 size);
 void *AllocFromHeapAtEnd(u32 heap_id, u32 size);
 void FreeToHeap(void *ptr);
-void FUN_02016A8C(u32 heap_id, void * ptr);
+void FreeToHeapExplicit(u32 heap_id, void * ptr);
 THUMB_FUNC u32 FUN_02016AF8(u32 param0);
```

Comment:
can you clean this header up too, I know you didn't do it but that would be appreciated